cellar_rebalancer/src/gas.rs

```diff
@@ -1,15 +1,13 @@
 //! Gas models
+#[allow(unused_imports)]
```

Comment:
Why allow unused imports here? I don't see any commented out code or anything that is "using" the unused imports.